.github/workflows/validate-commit-messages.yml

```
@@ -0,0 +1,45 @@
+name: Validate Commit Messages
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)